app/views/users/profile.html.erb

```diff
@@ -69,9 +69,14 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
+    <%if (@profile_user.followers.count>0)%>
```

Comment:
@shubhangi013 The if statements don’t need brackets around them. Also add spaces after `<%` and before the `%>`.

Reply:
Will do that! Thanks